test/unit/uri.c

```diff
@@ -1,3 +1,5 @@
+#define UNIT_TAP_COMPATIBLE 1
+#include "unit.h"
```

Comment:
It seems, this commit is not needed now, see 62aed2cf147f2ca24a34065219aa957f71194a52.